client/src/pages/private/ParticipantTableDialogues.js

```diff
@@ -56,8 +56,8 @@ export const ParticipantTableDialogues = ({
     >
       {activeModalForm === 'single-select-site' && (
         <SelectProspectingSiteForm
-          initialValues={{ prospectingSites: [] }}
-          validationSchema={ProspectingSitesSchema}
+          initialValues={{ prospectingSite: undefined }}
```

Comment:
```suggestion
          initialValues={{ prospectingSite: '' }}
```
`prospectingSite` is a string value, so please initiate with an empty string.

Reply:
`prospectingSite` is a number, form saves siteId as a value

Reply:
Then use negative numbers and update the schema so it can only accept positive values .....

Reply:
ok, then no change required